### PR TITLE
version 1.0.0

### PR DESCRIPTION
**Pull request by malaffoon**
The CI process is such that develop should have the version number set to what we're working towards, in this case 1.0.0.

I've already bumped common and schema.

Note that i am not resetting the build number cause it really should've been 1.0.0 for a while.

### CODE REVIEW

File: webapp/src/main/webapp/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "client",
-  "version": "0.0.0",
+  "version": "1.0.0",
```
Comment (malaffoon) [inline webapp/src/main/webapp/package.json]: Is this right? Is the client version used/shown anywhere?

Comment (chdietz) [reply webapp/src/main/webapp/package.json]: Also wondering if there is a package-lock.json for RDW_Admin

Comment (rcordeau) [reply webapp/src/main/webapp/package.json]: Yes we need to update the package-lock.json file too.

Comment (chdietz) [reply webapp/src/main/webapp/package.json]: I do not see where this is displayed on the client.

Comment (rcordeau) [reply webapp/src/main/webapp/package.json]: I don't think it is displayed, but we should probably be consistent.

Comment (bgsavage) [reply webapp/src/main/webapp/package.json]: do we want to give an actual `name` at this point as well?

Comment (rcordeau) [reply webapp/src/main/webapp/package.json]: How about testApp1?

Comment (bgsavage) [reply webapp/src/main/webapp/package.json]: I prefer `robTestApp14`.  Shows effort.

**Pull request merged**